pkg/driver/helmdriver.go

```diff
@@ -69,13 +72,37 @@ func (d *helmDriver) Install(ctx context.Context,
 	if err != nil {
 		return fmt.Errorf("loading helm chart %s: %w", name, err)
 	}
+	// If no target namespace provided read chart values to find namespace
+	if namespace == "" {
+		val, exists := helmChart.Values["defaultNamespace"]
+		if exists {
+			namespace = val.(string)
+		} else {
+			// Fall back case of assuming its default
+			namespace = "default"
+		}
+	}
```

Comment:
```suggestion
	namespace := "default"
	if chartNS, ok := helmChart.Values["defaultNamespace"]; ok {	   
	   namespace = chartNS.(string)
	}
```
Shorter, and more idiomatic.